flink/v1.13/flink-runtime/NOTICE

```diff
@@ -89,3 +89,14 @@ file:
 |
 | This product includes software developed at
 | The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+This binary artifact includes Apache HTTPComponents Client with the following
+in its NOTICE file:
+
+| Apache HttpComponents Client
+| Copyright 1999-2021 The Apache Software Foundation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
```

Comment:
[question] Looks like they haven't updated the copyright to 2022 : https://github.com/apache/httpcomponents-client/blob/master/NOTICE.txt . (for year 2021 they did update this at [Feb 2021](https://github.com/apache/httpcomponents-client/commit/59491adbdb06f4e896a1659be9f2dda8a3b8fdb8))
would we be fine using this notice in 2022 ?

Reply:
Yeah we’d be fine. Two reasons:
1. Copyright technically only matters from the starting year. People update them every year for the new year generally because they feel they should (and 100% I am the one to have updated the Iceberg files for the new year the past 2 years so I am guilty of this too), but it’s not actually required. The copyright only applies based on the start date.
2. This is the file they have presently :)

But reason 1 is good to know.

If you want, they might accept a PR to update their notice though 😄

Reply:
Thank you @kbendick,  your rationale makes sense, added the PR for the same.